.gitignore

```diff
@@ -12,6 +12,7 @@ envs/server
 /.pnp
 .pnp.js
 
+/envs/server
```

Comment:
we should just ignore all .env to avoid any accidental commits of any of those files. 

There could be a `dev.env.sample` file which can hold the template.